### PR TITLE
[14.0][MIG] stock_demand_estimate

### PR DESCRIPTION
**Pull request by DavidBForgeFlow**
Standard migration to v.14

### CODE REVIEW

**Comment by rousseldenis** [comment]
@DavidBForgeFlow Thanks for this. Could you add some tests for uncovered lines? And could you change your commit message as there is no "Migration to ... "

File: stock_demand_estimate/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016-20 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Stock Demand Estimate",
+    "summary": "Allows to create demand estimates.",
+    "version": "14.0.1.1.0",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "development_status": "Production/Stable",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
```
Comment (rousseldenis) [inline stock_demand_estimate/__manifest__.py]: Please change this to 'Warehouse'

**Review by hparfr — APPROVED** [review]

File: stock_demand_estimate/__manifest__.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply stock_demand_estimate/__manifest__.py]: @DavidBForgeFlow ping

Comment (DavidBForgeFlow) [reply stock_demand_estimate/__manifest__.py]: Changed!

**Review by LoisRForgeFlow — APPROVED** [review]
Tested locally, work fine :+1:

**Pull request merged**